src/pages/MetricDetail.svelte

```diff
@@ -10,7 +10,7 @@
   import { pageTitle } from "../state/stores";
 
   export let params;
-  let metricName = params.metric.replaceAll("-", ".");
+  let metricName = params.metric;
```

Comment:
I think we should just take out this variable and replace it with "params.metric" everywhere. I think `metricName` falsely implies that this is the real name of the metric, rather than an identifier.

Reply:
Yep! Makes sense.